src/Mutation/NodeTraverserFactory.php

```diff
@@ -56,6 +56,10 @@ public function create(array $extraVisitors): PrioritizedVisitorsNodeTraverser
         $traverser = new PrioritizedVisitorsNodeTraverser(new NodeTraverser());
 
         $traverser->addPrioritizedVisitor(new NotMutableIgnoreVisitor(), 50);
+        $traverser->addPrioritizedVisitor(new NodeVisitor\NameResolver(null, [
```

Comment:
missing an import statement: we didn't use partial imports for the visitors in other places here